test/rekt/resources/kafkacat/configmap.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .name }}
+  namespace: {{ .namespace }}
+data:
+  payload: '{{ if .key }}{{ .key }}={{end}}{{ .payload }}'
```

Comment:
how/where is the configmap used?

Reply:
when Kafkacat is installed (https://github.com/knative-sandbox/eventing-kafka/pull/488/files#diff-c08ede98b036f16224e9ab7f7a22b8ac1e29498c62254c35df90cf854ed63d56R39), both the deployment and the configmap are created.